.github/workflows/golangci-lint.yml

```diff
@@ -1,4 +1,4 @@
-name: golangci-lint
+name: Linting with golangci-lint
```

Comment:
Could we leave this as `Linter` as we seem to be using nouns for other workflow names, i.e `CI Tests` and `TFLocal Instance`? Heck we can even rename the file to `lint.yml` as well.

Reply:
Ah, that's a fair point 😃. Looking now, I see your pending changes in #577 would alter this workflow to run gofmt as well, making the filename partially inaccurate. What do you think?